src/mplfinance/_arg_validators.py

```diff
@@ -110,14 +110,36 @@ def _mav_validator(mav_value):
     '''
     if isinstance(mav_value,int) and mav_value > 1:
         return True
-    elif not isinstance(mav_value,tuple) and not isinstance(mav_value,list):
+    elif not isinstance(mav_value,tuple) and not isinstance(mav_value,list) and not isinstance(mav_value,dict):
```

Comment:
When I first wrote this, I did not realize that `isinstance` can accept a tuple of type to indicate `or`, therefore this can be simplified to:
```
elif not isinstance(mav_value,(tuple,list,dict)):
```

Similar simplifications can be made below.  I would not have gone out of my way to make these simplifications, but since you are changing this code now anyway, it would be nice to simplify the code now.  Thank you.

Reply:
done